tests/models/swin/test_modeling_swin.py

```diff
@@ -493,6 +493,25 @@ def test_inference_image_classification_head(self):
         expected_slice = torch.tensor([-0.0948, -0.6454, -0.0921]).to(torch_device)
         self.assertTrue(torch.allclose(outputs.logits[0, :3], expected_slice, atol=1e-4))
 
+    @slow
+    def test_inference_interpolate_pos_encoding(self):
+        # ViT models have an `interpolate_pos_encoding` argument in their forward method,
+        # allowing to interpolate the pre-trained position embeddings in order to use
+        # the model on higher resolutions.
+        model = SwinModel.from_pretrained("microsoft/swin-tiny-patch4-window7-224").to(torch_device)
+
+        image_processor = self.default_image_processor
+        image = Image.open("./tests/fixtures/tests_samples/COCO/000000039769.png")
+        inputs = image_processor(images=image, size={"height": 480, "width": 480}, return_tensors="pt")
```

Comment:
Regarding @NielsRogge's comment - could we update these values to ones which aren't divisible by 32? This way we can make sure we're triggering the new code path. A good sense check would be to see if this fails if you set `interpolate_pos_encoding=False` in the forward call

Reply:
Make sense. I will update the size.